test/serializedobject-test.js

```diff
@@ -1,6 +1,19 @@
-var assert           = require('assert');
+'use strict';
+
+/* eslint-disable max-len*/
+/* eslint-disable key-spacing*/
+/* eslint-disable no-spaced-func*/
+/* eslint-disable space-before-blocks*/
+/* eslint-disable space-unary-ops*/
+/* eslint-disable no-void*/
+/* eslint-disable semi*/
+/* eslint-disable no-unused-vars*/
+/* eslint-disable quotes*/
```

Comment:
This is a single find and replace operation, it would have been faster to fix it than to type this disabling comment.